client/resource_group/controller/controller.go

```diff
@@ -1199,6 +1224,7 @@ func (gc *groupCostController) modifyTokenCounter(counter *tokenCounter, bucket
 		counter.notify.setupNotificationThreshold = 1
 		counter.notify.mu.Unlock()
 		counter.lastDeadline = deadline
+		gc.isThrottled.Store(true)
```

Comment:
Do we need to have a metric to record this status?

Reply:
added a log field in the debug log. Also, it's easy to be judged by the RU metrics.